scripts/issue/1122/create_gtc_position.py

```diff
@@ -0,0 +1,41 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+LOWER_RANGE_MULTIPLIER = 0.9
+UPPER_RANGE_MULTIPLIER = 2
+GTC_SELL_PCT = 0.16
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_uni_v3()
+
+    gtc = vault.contract(r.treasury_tokens.GTC)
+    weth = vault.contract(r.treasury_tokens.WETH)
+
+    vault.take_snapshot(tokens=[gtc, weth])
+
+    sell_amount = int(gtc.balanceOf(vault) * GTC_SELL_PCT)
+    vault.uni_v3.swap([gtc, weth], sell_amount)
+    vault.print_snapshot()
+
+    # hardcoded path else it will use the 1% fee pool with different rate
+    multihop_path = [gtc.address, 3000, weth.address]
+
+    gtc_rate = vault.uni_v3.get_amount_out(
+        [gtc, weth], 1e18, multihop_path=multihop_path
+    )
+
+    range_0 = gtc_rate * LOWER_RANGE_MULTIPLIER / 1e18
+    range_1 = gtc_rate * UPPER_RANGE_MULTIPLIER / 1e18
+
+    vault.uni_v3.mint_position(
+        r.uniswap.v3pool_eth_gtc,
+        range_0,
+        range_1,
+        weth.balanceOf(vault),
+        gtc.balanceOf(vault),
```

Comment:
this will need to be done dusty, since outcome of swap is not 100% deterministic in our case

Reply:
good catch, using `0.97`: 517e2e9

Reply:
should not be only dusty for weth?

Reply:
yup, dont see why its needed for gtc

Reply:
> should not be only dusty for weth?

e5833d1